packages/config/src/android/Orientation.ts

```diff
@@ -7,6 +7,14 @@ export function getOrientation(config: ExpoConfig) {
   return typeof config.orientation === 'string' ? config.orientation : null;
 }
 
+export function prepareOrientation(orientation: string) {
+  if (orientation === 'default') {
```

Comment:
maybe it's a stupid question, but why you are handling 'default' value differently than missing one?

Reply:
Just because the missing one is already handled when called this function (line 7)